.github/workflows/pipeline.yaml

```diff
@@ -99,6 +99,44 @@ jobs:
         if: always()
         run: docker stop ${{ steps.start_container.outputs.container_id }}
 
+  test-with-k3s:
```

Comment:
Note to self: Move after build-push-edge-endpoint-multiplatform once this PR is ready to merge